src/main/java/de/tum/in/www1/artemis/web/rest/dto/LectureUnitInformationDTO.java

```diff
@@ -0,0 +1,6 @@
+package de.tum.in.www1.artemis.web.rest.dto;
+
+import java.util.List;
+
+public record LectureUnitInformationDTO(List<LectureUnitSplitDTO> units, Integer numberOfPages, Boolean removeBreakSlides) {
```

Comment:
```suggestion
public record LectureUnitInformationDTO(List<LectureUnitSplitDTO> units, Integer numberOfPages, boolean removeBreakSlides) {
```
Can this be a primitive type? This could simplify some checks.

Reply:
yes, i changed to primitive type